jenkins/internal-ci/generic/pr/cortx_deployment/cortx-rgw.groovy

```diff
@@ -76,7 +77,7 @@ pipeline {
                 """
 
                 dir('cortx-re') {
-                    checkout([$class: 'GitSCM', branches: [[name: '*/main']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'CloneOption', depth: 1, honorRefspec: true, noTags: true, reference: '', shallow: true], [$class: 'AuthorInChangelog']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: 'https://github.com/Seagate/cortx-re']]])
+                    checkout([$class: 'GitSCM', branches: [[name: '*/pr-sha']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'CloneOption', depth: 1, honorRefspec: true, noTags: true, reference: '', shallow: true], [$class: 'AuthorInChangelog']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: 'https://github.com/nitisdev/cortx-re']]])
```

Comment:
Please change this back to cortx-re repo details

Reply:
Done